### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PotatoQuality**
Update with the brand new features of v4 👍 
- Removed v2 from the table
- added v4
- added new notifications in the notifications list
- moved feature requests above

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,31 +94,27 @@ We have already done most of the work, but some features are still be missing in
...
+| Service  | Version 4 (Beta) | Version 3 (Stable)|
 |----------|:----------:|:----------:|
-| Multiple Plex/Emby Servers| Yes | No |
-| Emby & Plex support | Yes | Yes |
-| Mono dependency | No | Yes |
-| Notifications support | Yes| Yes |
-| Landing page | Yes (brand new) | Yes |
+| Multiple Plex/Emby/Jellyfin Servers | Yes | Yes |
+| Emby/Jellyfin & Plex support | Yes | Yes |
+| Mono dependency | No | No |
```
Comment (tidusjar) [inline README.md:101]: Probably remove this now

File: README.md [hunk 2]
```diff
@@ -83,31 +94,27 @@ We have already done most of the work, but some features are still be missing in
...
-
-# Feature Requests
-Feature requests are handled on Feature Upvote.
-
-Search the existing requests to see if your suggestion has already been submitted.
-(If a similar request exists, please vote, or add additional comments to the request)
-
-#### [![Feature Requests](https://cloud.githubusercontent.com/assets/390379/10127973/045b3a96-6560-11e5-9b20-31a2032956b2.png)](https://features.ombi.io)
+| DogNzb | Yes | Yes |
+| Headphones | Not yet | Yes |
```
Comment (tidusjar) [inline README.md:116]: Put `NO` as there are no plans to support `Headphones`

File: README.md [hunk 3]
```diff
@@ -57,23 +65,26 @@ Here are some of the features Ombi V3 has:
...
+* Sonarr V2 and V3
+* Radarr V2
 * Lidarr
 * DogNzb
 * Couch Potato
 
 
 ### Notifications
 Supported notifications:
+* Mobile and legacy mobile
```
Comment (tidusjar) [inline README.md:78]: Just Mobile is fine

File: README.md (hunk 1 above)
Comment (PotatoQuality) [reply README.md:101]: fixed

File: README.md (hunk 2 above)
Comment (PotatoQuality) [reply README.md:116]: fixed

File: README.md (hunk 3 above)
Comment (PotatoQuality) [reply README.md:78]: fixed

**Comment by tidusjar** [comment]
Thanks!

**Pull request merged**